modules/ROOT/pages/install-self-managed-network-configuration.adoc

```diff
@@ -42,12 +42,25 @@ To function correctly, Runtime Fabric requires the following hostname configurat
 * US control plane: `transport-layer.prod.cloudhub.io`
 * EU control plane: `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.
 | 443 (v1.8.50 and later)| TCP (Lumberjack) a| 
-* US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io`
-* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
-* EU control plane: `dias-ingestor-router.eu-central-1.prod-eu.msap.io` | Anypoint Monitoring agent for Runtime Fabric.
+* US control plane: +
+`dias-ingestor-router.us-east-1.prod.cloudhub.io` +
+`us1.ingest.mulesoft.com/api/v2/lmetrics` +
+`us1.ingest.mulesoft.com/api/v2/metrics` +
+`us1.ingest.mulesoft.com/api/v2/traces` +
+`us1.ingest.mulesoft.com/api/v2/logs` +
+
+* EU control plane: +
+`dias-ingestor-nginx.prod-eu.msap.io` +
+`dias-ingestor-router.eu-central-1.prod-eu.msap.io` +
+`eu1.ingest.mulesoft.com/api/v2/lmetrics` +
+`eu1.ingest.mulesoft.com/api/v2/metrics` +
+`eu1.ingest.mulesoft.com/api/v2/traces` +
+`eu1.ingest.mulesoft.com/api/v2/logs` +
+
+ | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.
 | 5044 (legacy) |TCP (Lumberjack) a| 
 * US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
-* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io` | Anypoint Monitoring agent for Runtime Fabric.
+* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io` | xref:use-anypoint-monitoring.adoc#enable-anypoint-monitoring[Anypoint Monitoring] agent for Runtime Fabric.
```

Comment:
@bin-shi-mulesoft Should we replace this one dias-ingestor-nginx.prod-eu.msap.io with another endpoint?

Reply:
@kevintroller No, because this is the legacy hostname.